lib/features/evaluate/config.dart

```diff
@@ -149,6 +150,41 @@ class EvaluateConfigState extends ConsumerState<EvaluateConfig> {
     ),
   ];
 
+  Map<String, String> datasetTypes = {
+    'Training': '''
+
+    Evaluate the model using the training dataset.
+    This will give an optimistic estimate of the 
+    performance of the model.
+
+    ''',
+    'Validation': '''
```

Comment:
Could we use Tuning here rather than Validation. And then from what the user sees is Tuning for now and then update it to be Tuning or Validation (another issue?) depending on `useValidationSettingProvider`

Reply:
Done